src/main/java/io/kokuwa/maven/helm/DependencyUpdateMojo.java

```diff
@@ -0,0 +1,48 @@
+package io.kokuwa.maven.helm;
+
+import java.nio.file.Path;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import lombok.Setter;
+
+/**
+ * Mojo for executing "helm dependency-update".
+ *
+ * @see <a href="https://helm.sh/docs/helm/helm_dependency_update/">helm dependency-update</a>
+ * @since 6.10.2
+ */
+@Mojo(name = "dependency-update", defaultPhase = LifecyclePhase.PROCESS_RESOURCES, threadSafe = true)
+@Setter
+public class DependencyUpdateMojo extends AbstractDependencyMojo {
+
+	/**
+	 * Set this to <code>true</code> to skip invoking dependency-update goal.
+	 *
+	 * @since 6.10.2
+	 */
+	@Parameter(property = "helm.dependency-update.skip", defaultValue = "true")
```

Comment:
A/Q: I wonder if we should just not bind it to a lifecycle phase.  Obviously this makes it a bit easier to enable, but if we expect usage to be _either_ build _or_ update then it might make sense to have users enable it by binding it to a phase rather than setting a property to true.  I could see it going either way, and I don't think there are major ramifications either way.

Reply:
Spoke with Carter offline.  This Mojo is already excluded from the default plugin execution and needs to be manually added to actually be called.  Because of this, we decided to switch the default value of this skip property to `false`, as it will only ever be called if the user explicitly added it to their POM.